consultation_analyser/consultations/views/answers.py

```diff
@@ -50,3 +53,100 @@ def index(
     }
 
     return render(request, "consultations/answers/index.html", context)
+
+
+@login_required  # TODO: check if we still need this
```

Comment:
We don't need the `login_required` decorator any more as we now use middleware that defaults to login required.